src/display/annotation_storage.js

```diff
@@ -0,0 +1,57 @@
+/* Copyright 2014 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+class AnnotationStorage {
+  constructor(storage = {}) {
+    this._storage = storage;
+  }
+
+  /**
+   * Get the value for a given key if it doesn't exist
+   * or store and return the defaultValue
+   *
+   * @public
+   * @memberof AnnotationStorage
+   * @param {String} key
```

Comment:
`String` -> `string` since `typeof 'foo' === 'string'` (same for the method below)